priv/libexec/commands/win/console.ps1

```diff
@@ -18,6 +18,6 @@ $post_start = {
 }
 
 # Run post-start hooks asynchronously
-start-job -Name "post_start hooks" -ScriptBlock $post_start
+start-job -Name "post_start hooks" -ScriptBlock $post_start | out-null # hide the output from start-job
 
-& $werl @argv
+start-process "$erl" -ArgumentList "$argv" -Wait -NoNewWindow #execute the application in the current shell window
```

Comment:
My impression was that `werl` is the only "true" Erlang shell, the command-line version is considerably simpler, missing many of the useful features - has that changed? I would expect someone would want the proper shell when attaching a console in production. Thoughts?

Reply:
possibly I'm incorrect but I was under the assumption that the two were identical, in the case that they are not, it may be a good idea to have an optional config flag